compute-pcrs/Containerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker.io/library/rust:1.89-trixie AS builder
+WORKDIR /compute-pcrs
+COPY Cargo.toml Cargo.lock .
+COPY compute-pcrs compute-pcrs
+# Hack: Set compute-pcrs as sole member to avoid needing to copy other crates.
+# In that case, a rebuild would be triggered upon any change in those crates.
+RUN sed -i 's/members =.*/members = ["compute-pcrs"]/' Cargo.toml && \
+    git clone --depth 1 https://github.com/confidential-clusters/reference-values && \
+    cargo build --release -p compute-pcrs
+
+FROM docker.io/library/debian:trixie
+COPY --from=builder /compute-pcrs/target/release/compute-pcrs /usr/local/bin
+COPY --from=builder /compute-pcrs/reference-values /reference-values
+ENTRYPOINT ["compute-pcrs", \
+    "--efivars", "/reference-values/efivars/qemu-ovmf/fcos-42", \
+    "--mokvars", "/reference-values/mok-variables/fcos-42"]
```

Comment:
nit: usually the args are specified at container creation

Reply:
Kubernetes overwrites the entrypoint in any case. What I don't like is that there is fcos-42 hardcoded there

Reply:
Has a further decision been taken on this topic? I see this is one of the only references to `reference-values/mok-variables/fcos-42` which I would like to update to `reference-values/mok-variables/fedora-42` for the sake of simplicity.

Are we safe if I move https://github.com/confidential-clusters/reference-values/pull/3 on?